packages/tests/tests/filter.node.spec.ts

```diff
@@ -115,3 +115,107 @@ describe("Waku Filter", () => {
     expect(messageCount).to.eq(1);
   });
 });
+
+describe("Waku Filter: V2", () => {
+  let waku: LightNode;
+  let nwaku: Nwaku;
+
+  afterEach(async function () {
+    !!nwaku &&
+      nwaku.stop().catch((e) => console.log("Nwaku failed to stop", e));
+    !!waku && waku.stop().catch((e) => console.log("Waku failed to stop", e));
+  });
+
+  beforeEach(async function () {
+    this.timeout(15000);
+    nwaku = new Nwaku(makeLogFileName(this));
+    // New Filter protocol only works on go-waku right now
+    if (nwaku.nodeType !== "go-waku") this.skip();
```

Comment:
Is filter ready in nwaku master HEAD? (cc @jm-clius) if so, it'd be good to test against it now, before a release.

Reply:
filter service node is ready, yes.